torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -910,6 +910,18 @@ def _is_packable_linear(match):
         Check if the node is supported for MKLDNN linear.
         """
         linear_node = match.output_node()
+        # mkldnn linear only supports beta=1or0 and alpha=1
+        if linear_node.target == aten.addmm.default:
+            alpha = (
+                None
+                if "alpha" not in linear_node.kwargs
+                else linear_node.kwargs["alpha"]
```

Comment:
Try `linear_node.kwargs.get("alpha", 1.0)`, it may provide a default value when not exist.

Reply:
Thanks and modified.